packages/mpt/src/mpt.ts

```diff
@@ -218,9 +219,10 @@ export class MerklePatriciaTrie {
 
     // If value is empty, delete
     if (value === null || value.length === 0) {
-      return this.del(key)
+      await this.del(key)
```

Comment:
This initially returned from this method, but now it does not, so I suspect this will yield some bugs (?)